.gitignore

```diff
@@ -0,0 +1,2 @@
+*test.md
```

Comment:
- [ ] You added test.md in gitignore but this not how it should be. Please read https://www.atlassian.com/git/tutorials/saving-changes/gitignore#:~:text=If%20you%20want%20to%20ignore,directory%20as%20an%20ignored%20file.